packages/terra-cell-grid/src/terra-dev-site/doc/example/CellGridExample.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import CellGrid, {
+  Cell,
+} from 'terra-cell-grid'; // eslint-disable-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
+
+const CellGridExample = () => (
+  <CellGrid>
```

Comment:
Looking at this, I feel like the names <GridRow> and <GridCell> would make sense here..

Will these components be expected to be used in terra-grid at all? I'm assuming no?

Reply:
I like the current names :)